install-config.yaml

```diff
@@ -3,13 +3,13 @@
 apiVersion: v1
 baseDomain: testing
 compute:
-- architecture: amd64
+- architecture: 
```

Comment:
I think we leave default values in `install-config.yaml`, so you can keep the `amd64` there.